test/tests/data/search/a.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>hello</title>
+  </head>
+  <body>
+    <p>mjktkiuewf</p>
```

Comment:
I actually did prefer something more... legible. Makes it easier to ensure that there are no typos, etc. Currently resetting DB is not working, but there are no conflicts yet either. I'm sure we'll figure out how to fix the DB resetting soon and we'll patch this up at that time. For now, I think foo bar baz is good without DB reset.